recipes/brian2genn/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "brian2genn" %}
+{% set version = "1.6" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/Brian2GeNN-{{ version }}.tar.gz
+  sha256: f0c42e5fe3ecfd6d373353caa5a431e13284288b07340e88f1c2829f4de53965
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+    - setuptools
+  run:
+    - brian2 >=2.4.2
+    - python >=3.6 
```

Comment:
Looks like this depends on `six` as well

Reply:
I swear that wasn't there when I looked. Apparently I need either more coffee, or less. 🐑